outputs/pagerduty.go

```diff
@@ -0,0 +1,74 @@
+package outputs
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewPagerdutyClient returns a new output.Client for accessing the Pagerduty API.
+func NewPagerdutyClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+	if len(config.Pagerduty.Assignee) > 0 && config.Pagerduty.EscalationPolicy != "" {
+		return nil, fmt.Errorf("assignee and escalation policy cannot be both configured")
+	}
+
+	return &Client{
+		OutputType:      "GCP",
+		Config:          config,
+		Stats:           stats,
+		PromStats:       promStats,
+		StatsdClient:    statsdClient,
+		DogstatsdClient: dogstatsdClient,
+		PagerdutyClient: pagerduty.NewClient(config.Pagerduty.APIKey),
+	}, nil
+}
+
+// PagerdutyPost posts incident to Pagerduty
+func (c *Client) PagerdutyPost(falcopayload types.FalcoPayload) {
```

Comment:
```suggestion
func (c *Client) PagerdutyCreateIncident(falcopayload types.FalcoPayload) {
```

For following their SDK